### PR TITLE
fix(remount): return immediately if volume is not ready for mounting

### PR DESCRIPTION
**Pull request by payes**
This PR returns the Remount call immediately if volume
is not available/reachable. Currently it waits for 12 seconds
if volume is not ready and this will cause delay in mounting
the same application volume if application pod deleted/scaleup/scaledown.
Signed-off-by: Payes Anand <payes.anand@mayadata.io>

### CODE REVIEW

File: pkg/driver/mount.go [hunk 1]
```diff
@@ -293,17 +313,13 @@ func (n *NodeMounter) remountVolume(
...
-	_, err = waitForVolumeToBeReady(vol.Name, n.client)
-	if err != nil {
-		return
-	}
 
-	err = waitForVolumeToBeReachable(fmt.Sprintf("%v:%v", vol.Spec.ISCSISpec.TargetIP,
-		vol.Spec.ISCSISpec.TargetPort))
-	if err != nil {
-		return
+	if ready, err := isVolumeReady(vol.Name, n.client); err != nil || !ready {
```
Comment (mittachaitu) [inline pkg/driver/mount.go:317]: Q: When will the JivaVolume phase and status will be updated?

**Review by mittachaitu — APPROVED** [review]
Changes are good

File: pkg/driver/mount.go (hunk 1 above)
Comment (payes) [reply pkg/driver/mount.go:317]: This will be updated only when 2 replicas are connected and in RW state

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**